src/models/global.rs

```diff
@@ -150,20 +172,33 @@ impl Default for V4L2Standard {
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize, Validate)]
 #[serde(default, rename_all = "camelCase", deny_unknown_fields)]
 pub struct GrabberV4L2 {
+    pub enable: bool,
     pub device: String,
     pub input: i32,
     pub standard: V4L2Standard,
     pub width: u32,
     pub height: u32,
+    pub encoding: String,
+    pub flip: String,
```

Comment:
This should be an enum, like it is done for the standard field.

Reply:
Yep, like above: https://github.com/hyperion-project/hyperion.ng/blob/master/include/utils/PixelFormat.h#L151

```rs
#[derive(Debug, Clone, Copy, PartialEq, Serialize, Deserialize)]
#[serde(rename_all = "lowercase", deny_unknown_fields)]
pub enum FlipMode {
    #[serde(rename = "NO_CHANGE")]
    NoChange,
    Horizontal,
    Vertical,
    Both,
}
```